src/resources/craftsmen.csv

```diff
@@ -0,0 +1,3 @@
+id:1,name:bob,surname:smith
```

Comment:
your data format in CSV file  is incorrect you shouldn't have a key-value format for each field but you have to write data with a header as a first line and data in others 
![image](https://github.com/evTyurin/SENLA-java-courses/assets/39222770/cd5f9625-7767-4982-9e8c-bdf9f10a5bb0)